doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -8,11 +8,13 @@ Ltac2
 General design
 --------------
 
-There are various alternatives to Ltac1, such as Mtac or Rtac for instance.
-While those alternatives can be quite different from Ltac1, we designed
-Ltac2 to be as close as reasonably possible to Ltac1, while fixing its
-:ref:`defects <ltac_defects>`.
+Ltac2 is a tactics metalanguage for Rocq. It is mostly used to write new
+tactics using a mix of :cmd:`Ltac2` definitions and :cmd:`Ltac2 Notation`\s.
+Ltac2 provides an API in the various modules of the Ltac2 library, see
+`<https://rocq-prover.org/corelib/index.html#Ltac2>`_.
```

Comment:
This should be a relative link, so that the right version is linked. This also means that the link will start working right away for the master version of the doc after this is merged.
```suggestion
`the relevant section of Corelib's documentation <../../corelib/index.html#Ltac2>`_.
```